packages/@aws-cdk/aws-cloudfront-origins/lib/s3-origin.ts

```diff
@@ -58,7 +58,9 @@ class S3BucketOrigin extends cloudfront.OriginBase {
 
   public bind(scope: cdk.Construct, options: cloudfront.OriginBindOptions): cloudfront.OriginBindConfig {
     if (!this.originAccessIdentity) {
-      this.originAccessIdentity = new cloudfront.OriginAccessIdentity(scope, 'S3Origin');
+      this.originAccessIdentity = new cloudfront.OriginAccessIdentity(scope, 'S3Origin', {
+        comment: `Access identity for ${options.originId}`,
```

Comment:
Minor suggestion

```suggestion
        comment: `Access identity for origin [${options.originId}]`,
```

Reply:
This property is already super-long in the console experience, and the IDs all have 'origin' in them by definition, so I would rather opt for the more succinct version in this case.